src/app/public/modules/toast/toast.module.ts

```diff
@@ -63,8 +59,7 @@ import {
   ],
   providers: [
     SkyAppWindowRef,
-    SkyToastService,
-    SkyToastAdapterService
+    SkyToastService
```

Comment:
`SkyToastService` can be removed and `providedIn` root?

Reply:
Oops, good catch. Fixed!